### PR TITLE
Package tvb-rest-client to pypi 

### PR DESCRIPTION
**Pull request by gabiflorea10**

### CODE REVIEW

**Comment by gabiflorea10** [comment]
> First commit refers to a package on pypi test.

File: tvb_build/package_for_pip_client.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+for pipPackage in "${folders2pack[@]}"; do
+
+    echo "============================="
+    echo " Packing: " $pipPackage
+    echo "============================="
+
+    cd $pipPackage
+    python setup_client.py sdist
+    python setup_client.py bdist_wheel
```
Comment (liadomide) [inline tvb_build/package_for_pip_client.sh]: These 2 lines I expect, but the loop over folders2pack does not make any sense.
Have you considered to simply add the setup_client.py use in the already existent SH (package_for_pip) ?

File: framework_tvb/setup_client.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import os
+import shutil
+import setuptools
+
+
+VERSION = "2.0a1"
+
+TVB_TEAM = "Mihai Andrei, Lia Domide, Stuart Knock, Bogdan Neacsa, Paula Sansz Leon, Marmaduke Woodman"
+
+TVB_INSTALL_REQUIREMENTS = ["allensdk", "BeautifulSoup4", "cherrypy", "flask", "flask-restplus", "formencode", "Jinja2",
```
Comment (liadomide) [inline framework_tvb/setup_client.py]: This list would be too long as a dependency path

File: framework_tvb/setup_client.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                            "h5py", "networkx", "nibabel", "numpy", "Pillow", "psutil", "scipy",
+                            "simplejson", "sqlalchemy", "sqlalchemy-migrate", "tvb-data", "tvb-gdist", "tvb-library"]
+
+with open(os.path.join(os.path.dirname(__file__), 'README.rst')) as fd:
+    DESCRIPTION = fd.read()
+
+setuptools.setup(name="tvb-rest-client-test27",
```
Comment (liadomide) [inline framework_tvb/setup_client.py]: Have you considered adding this to the existing setup.py ?

File: tvb_build/package_for_pip_client.sh (hunk 1 above)
Comment (gabiflorea10) [reply tvb_build/package_for_pip_client.sh]: That's true. The loop over folders2pack should be removed. 
For second question... Yes, I took into consideration this solution too, but I think that probably we want to package the rest-client independently of the entire framework and having two different SH for packaging allow us to do this.

File: framework_tvb/setup_client.py (hunk 3 above)
Comment (gabiflorea10) [reply framework_tvb/setup_client.py]: As I aforementioned, I followed the idea of having a clear separation between framework packaging and rest-client packaging. 
But, If we prefer to have all packaging setups together, we can try to do this.
Also, for the previous message related to SH, we can try to put all together, if this is the preferred way.

File: framework_tvb/setup_client.py (hunk 2 above)
Comment (gabiflorea10) [reply framework_tvb/setup_client.py]: Yes, it should be refactored.

File: tvb_build/package_for_pip_client.sh (hunk 1 above)
Comment (liadomide) [reply tvb_build/package_for_pip_client.sh]: We rather want to have one single Jenkins build doing all Pypi packages. With any new release in the future I hope we can release everything at the same time

File: framework_tvb/setup_client.py (hunk 3 above)
Comment (liadomide) [reply framework_tvb/setup_client.py]: Yes, I would prefer to put things together

**Comment by liadomide** [comment]
I close this because Paula took over and did the changes in tvb-rest directly

**Pull request closed without merging**